libs/dh/shared/feature-flags/src/lib/dh-feature-flags.ts

```diff
@@ -73,6 +73,10 @@ export const dhFeatureFlagsConfig = {
       DhAppEnvironment.prod,
     ],
   },
+  'metering-points-master-data-upload': {
+    created: latestBump,
+    disabledEnvironments: [DhAppEnvironment.preprod, DhAppEnvironment.prod],
```

Comment:
it should be disabled for environments that are connected to DH2, so Test should be disabled